ci-operator/config/openshift/agent-installer-utils/openshift-agent-installer-utils-main.yaml

```diff
@@ -0,0 +1,54 @@
+base_images:
+  base:
+    name: "4.13"
+    namespace: ocp
+    tag: base
+  golangci-lint:
+    name: golangci-lint
+    namespace: ci
+    tag: v1.50.1
+build_root:
+  from_repository: true
+promotion:
```

Comment:
Note: for this we'll need an `image` section building a `agent-installer-utils` image using a Dockerfile.ocp stored in the repo (it could be done later)

Reply:
so I'll drop it until we have it